# 制御周波数[Hz]
hz: 10  # ループ周波数 [Hz]

# frame_id
node_frame: "base_footprint"  # ノード
path_frame: "odom"            # target_path
goal_frame: "odom"            # local_goal

# ノード探索の終了判定
goal_tolerance: 1.0  # local_goal_に対する許容誤差 [m]
finish_dist: 2.0     # ロボットからノード先端までの距離 [m]
# finish_dist: 4.0     # ロボットからノード先端までの距離 [m]

# glocal_goal
dist_to_update_glocal_goal: 1.5  # glocal_goalを更新する基準となるロボットとglocal_goalの距離 [m]

# 速度関連
max_vel: 1.2      # 最高並進速度 [m/s]
max_yawrate: 4.5  # 最高旋回速度 [rad/s]
max_speed: 3.0    # タイヤの最高回転速度 [m/s]
speed_reso: 0.1    # 速度を探索する際の刻み幅 [m/s]

# CCV関連
max_steer_angle: 20  # ステア角の最大値 [deg]
steer_angle_reso: 1.0 # ステア角を探索するときの刻み幅 [deg]
tread: 0.5           # トレッド [m]

# path関連
path_reso: 0.05   # 生成するpathの刻み幅 [m]
theta_reso: 0.01  # 候補となるpathを生成する際の方位の刻み幅 [rad]
search_step: 10    # 何ステップ先まで評価値を計算するか

# 評価関数関連
# weight_heading: 0.4          # 評価関数1項目　重みづけ定数
# weight_glocal_heading: 0.2   # 評価関数2項目　重みづけ定数
weight_heading: 0.3          # 評価関数1項目　重みづけ定数
weight_glocal_heading: 0.6   # 評価関数2項目　重みづけ定数
weight_cost_map: 1.5         # 評価関数3項目　重みづけ定数
min_cost: 50.0               # 割り当てるコストの最小値（cost_map_creatorのyaml参照）